packages/type/scss/font-face/_mono.scss

```diff
@@ -11,420 +11,1156 @@
 /// @access public
 /// @group @carbon/type
 @mixin carbon--font-face-mono {
-  // .woff support for IE11
   @font-face {
     font-display: $carbon--font-display;
     font-family: 'IBM Plex Mono';
-    font-style: italic;
-    font-weight: 300;
-    src: local('IBM Plex Mono Light Italic'), local('IBMPlexMono-LightItalic'),
-      url(https://fonts.gstatic.com/s/ibmplexmono/v4/-F6sfjptAgt5VM-kVkqdyU8n1ioSflV1gMoW.woff)
+    font-style: normal;
+    font-weight: 700;
+    src: local('IBM Plex Mono Bold'), local('IBMPlexMono-Bold'),
+      url('fonts/IBM-Plex-Mono/fonts/complete/woff2/IBMPlexMono-Bold.woff2')
+        format('woff2'),
```

Comment:
I think since this is the IE11 fallback we would just use the woff one instead of loading in woff2. I believe the rationale for this is that if they can't load the unicode-range versions it would fall back to this woff version, but if the woff2 is specified it may load that instead of the splits (not 100% sure though)

Reply:
@joshblack If that's the case, should we update this in the Plex repo? What I've done here is taken the [direct output](https://1.www.s81c.com/common/carbon/plex/mono.css) that is on the server and copied it over.

IIUC, [the repo](https://github.ibm.com/webstandards/plex-cdn) behind this is simply spitting out the generated stuff from the Plex package without modification.

Reply:
Not sure if IE11 even matters anymore, it is reaching EOL in June and usage is incredibly low at this point.